charts/cronjob/templates/cronjob.yaml

```diff
@@ -185,12 +185,14 @@ spec:
             image: "{{ template "argonaut-deployment.imageName" . }}"
             imagePullPolicy: "Always"
```

Comment:
change this to IfNotPresent
(separate reason)

Reply:
Changed.